cms/djangoapps/contentstore/views/videos.py

```diff
@@ -46,6 +47,14 @@
     '.mov': 'video/quicktime',
 }
 
+THUMBNAIL_SUPPORTED_FILE_FORMATS = {
```

Comment:
If we will use profile image validator than this needs to be removed otherwise this needs to be carved if we do our own validation.